dayone.hs

```diff
@@ -21,28 +21,29 @@
 
 -- Part  1:
 -- change next line: add parentheses to make this equal "Functional"
-word1 = reverse "lanoi" ++ "tcnuF"    -- "++"concatenates concatenates strings
+word1 = reverse ("lanoi" ++ "tcnuF")    -- "++"concatenates strings
   
 
 incr n = n + 1 -- don't change this...
 
 -- Part 2:
 -- change next line to make equal 50
-fifty = incr 7 * 7
+fifty = incr (7 * 7)
 
 
 -- Part 3:
 -- this function adds 2 to its parameter. Change it to use `incr`
 -- twice, rather than using `+ 2`
-add2 n = n + 2
+add2 n = incr (incr n)
 
 
 -- Part 4:
 -- You could also write `add2` as:
 newAdd2 = incr . incr
 --
 -- Use the 'net to find out what the "." operator is called
--- and write it here: «your answer»
+-- and write it here: 
+-- Basically . applies functions one after the other, effectively composing them together. The example shows how to incr the result of the second incr.
 
```

Comment:
Exactly. You could also say that it creates a new function that combines the effect of applying the first to the result from the second, so f . g x is the same as f(g(x)) (in Java)